src/id/doujindesu/src/eu/kanade/tachiyomi/extension/id/doujindesu/DoujinDesu.kt

```diff
@@ -368,29 +377,63 @@ class DoujinDesu : ParsedHttpSource(), ConfigurableSource {
 
     override fun mangaDetailsParse(document: Document): SManga {
         val infoElement = document.select("section.metadata").first()!!
+        val AuthorName = when {
+            document.select(".metadata > table > tbody > tr > td:contains(Author) ~ td")
+                .isEmpty() -> "Tidak Diketahui"
+            else -> document.select(".metadata > table > tbody > tr > td:contains(Author) ~ td")
+                .joinToString { it.text() }
+        }
+        val GroupName = when {
+            document.select(".metadata > table > tbody > tr > td:contains(Group) ~ td")
+                .isEmpty() -> "Tidak Diketahui"
+            else -> document.select(".metadata > table > tbody > tr > td:contains(Group) ~ td")
+                .joinToString { it.text() }
+        }
+        val AuthorParser = when {
+            AuthorName.isNullOrEmpty() || AuthorName == "Tidak Diketahui" -> GroupName.takeIf { !it.isNullOrEmpty() && it != "Tidak Diketahui" } ?: "Tidak Diketahui"
+            else -> AuthorName
+        }
```

Comment:
I change how Author Section work, basicaly if AuthorName = null it change to GroupName, if both null it change to "Tidak Diketahui"